executor/executor.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 syzkaller project authors. All rights reserved.
+// Copyright 2020 syzkaller project authors. All rights reserved.
```

Comment:
Weird. Is it done by clang-format? But somehow it did not break all other *.h files, so I assume it should be fixable.

Reply:
Yes I think it is clang-format. I did not make any modifications to the Copyright statement other than the year.